source/fuzz/fuzzer.cpp

```diff
@@ -234,6 +235,8 @@ Fuzzer::Fuzzer(std::unique_ptr<opt::IRContext> ir_context,
   MaybeAddFinalPass<FuzzerPassAdjustLoopControls>(&final_passes_);
   MaybeAddFinalPass<FuzzerPassAdjustMemoryOperandsMasks>(&final_passes_);
   MaybeAddFinalPass<FuzzerPassAdjustSelectionControls>(&final_passes_);
+  MaybeAddFinalPass<FuzzerPassAddReadModifyWriteAtomicInstruction>(
```

Comment:
I don't think this should be a final pass.

Reply:
Done.
Could you please tell me why should change it to `RepeatedPass`? @paulthomson

Reply:
As I understood, fuzzer pass needs to unlock other passes to work appropriately also depends on other passes, but in our case, it does not depend on any transformation or passes, so I think it should be still `MaybeAddFinalPass`.
What do you think @paulthomson?

Reply:
There is no exact science, but fuzzer passes should probably be repeated passes, by default. Your pass that changes memory semantics is a good example of a pass that should be a final pass; if we ran such a pass repeatedly, we are likely to end up changing most (if not all) memory orders to sequentially consistent. This may end up decreasing our coverage of other memory orders, such as acquire, release, etc.

Reply:
Thanks for the clarification.

Reply:
Done.